yt/yt/ytlib/queue_client/registration_manager.cpp

```diff
@@ -398,7 +424,10 @@ void TQueueConsumerRegistrationManager::GuardedRefreshReplicationTableMappingCac
 
     for (const auto& replicatedTableInfo : replicatedTableMapping) {
         for (const auto& replica : replicatedTableInfo.GetReplicas()) {
-            ReplicaToReplicatedTable_[replica] = replicatedTableInfo.Ref;
+            ReplicaToReplicatedTable_[replica] = TResolvedObject{
+                .Path = replicatedTableInfo.Ref,
+                .ObjectType = replicatedTableInfo.ObjectType,
```

Comment:
If we implement it like this, replicas will have a type of `[Chaos]ReplicatedTable`, while the `[chaos]_replicated_table` objects themselves will have a `std::nullopt` type.

Ideally, we would want to separate these types, but that all depends on how this will be used in the end.